core/src/main/scala/org/apache/spark/deploy/worker/ui/LogPage.scala

```diff
@@ -137,8 +143,16 @@ private[ui] class LogPage(parent: WorkerWebUI) extends WebUIPage("logPage") with
       return ("Error: invalid log directory " + logDirectory, 0, 0, 0)
     }
 
+    // Find a log file name
+    val fileName = if (logType.equals("out")) {
```

Comment:
Do we need to include this in the try-catch block below as well?

Reply:
Which part are you worrying, @yaooqinn ?

Reply:
`normalizedLogDir.listFiles`?

Reply:
yes

Reply:
Got it. Let me move.